static/app/components/searchSyntax/renderer.tsx

```diff
@@ -67,6 +68,16 @@ function renderToken(token: TokenResult<Token>, cursor: number) {
   }
 }
 
+// XXX(epurkhiser): We have to animate `left` here instead of `transform` since
```

Comment:
You can use `display: inline-block` to get both inline wrapping behaviors and more efficient transform animations.